### PR TITLE
Add webrtc-adapter@3.0.0 with git auto-update

### LINKED ISSUE #9812: [Request] Add webrtc-adapter
**Library name:** webrtc-adapter
**Git repository url:** https://github.com/webrtc/adapter
**npm package url(optional):** https://www.npmjs.com/package/webrtc-adapter
**License(s):**
**Official homepage:**
**Wanna say something? Leave message here:**

=====================
Notes from cdnjs maintainer:
You are welcome to add a library via sending pull request,
it'll be faster then just opening a request issue,
and please don't forget to read the guidelines for contributing, thanks!!

### PR DESCRIPTION
**Pull request by clairetsai818**
close #9812, cc @KaptenJansson

Pull request for issue: #9812 

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/webrtc/adapter
 * License and its reference: BSD-3-Clause
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 669
   - Count of watchers: 84
   - Count of forks: 160

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by clairetsai818** [comment]
@PeterDaveHello Please help me review this, thanks.

File: ajax/libs/adapter/package.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
+{
+  "name": "adapter",
+  "keywords": [
```
Comment (PeterDaveHello) [inline ajax/libs/adapter/package.json]: 1. Do you create these keywords yourself? I think there are some official keywords: https://github.com/webrtc/adapter/blob/v3.0.0/bower.json#L19-L30

2. Let's put `keywords` a little bit lower place in `package.json`, reserve the upper place for those most important info.

**Comment by clairetsai818** [comment]
@PeterDaveHello I've corrected `keywords` part in package.json. Please help me review this, thanks.

**Comment by clairetsai818** [comment]
@PeterDaveHello I've corrected `keywords` part in package.json. Please help me review this, thanks.

**Comment by PeterDaveHello** [comment]
@clairetsai818 The library name was wrong, please don't change the library name by yourself without letting the others know next time.

**Pull request merged**